Web_Development/FrontEnd/HTML_Commands/HML_Commands.md

```diff
@@ -1 +1,83 @@
-hi
\ No newline at end of file
+### The html tag: 
+The <html> tag points that the code is written in html and </html> tag states that it's the end of html code.
+
```

Comment:
The sentence is unclear please rephrase it.

Reply:
done